camelot/view/proxy/collection_proxy.py

```diff
@@ -1225,10 +1226,23 @@ def get_admin( self ):
         self.logger.debug('get_admin called')
         return AdminRoute.admin_for(self.admin_route)
 
+    def add_action_route(self, action_route):
```

Comment:
is dit hier nt een beetje verwarrend ?

ik veronderstel dat hier de bedoeling is een onderscheid te maken tss de filters, wiens state enkel initieeel moet gezet w, en de actions wiens state moet geevalueerd w bij change van de selection ?

mss kunnen we dat beter doen dmv een atribuut op de action class of zo dat specifieert of de state moet worden geherevalueerd als de selectie zich aanpast ?

Reply:
Je bedoelt het verschil tussen add_action_route en set_action_routes?

Reply:
idd